src/provider.coffee

```diff
@@ -5,6 +5,25 @@ extensions        = require './extensions'
 
 
 
+paramMatches = (param, duplicate) ->
+  if Array.isArray param
```

Comment:
You know I never thought about throwing away the query params, duh. That is so much easier.